node/impl/full/eth.go

```diff
@@ -1167,13 +1164,9 @@ func (a *EthModule) EthTraceBlock(ctx context.Context, blkNum string) ([]*ethtyp
 }
 
 func (a *EthModule) EthTraceReplayBlockTransactions(ctx context.Context, blkNum string, traceTypes []string) ([]*ethtypes.EthTraceReplayBlockTransaction, error) {
-	if len(traceTypes) != 1 || traceTypes[0] != "trace" {
```

Comment:
@virajbhartiya why was this block removed? was this a mistake?

Reply:
Oh right that was a mistake, that wasn't supposed to be removed